src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java

```diff
@@ -87,23 +85,21 @@ private void processEnvelope(CloudBlobContainer blobContainer, Envelope envelope
         String containerName = blobContainer.getName();
         String zipFileName = envelope.getZipFileName();
         UUID envelopeId = envelope.getId();
-        CloudBlockBlob blobClient = null;
+        CloudBlockBlob blob = null;
+        Optional<String> leaseId = Optional.empty();
 
         try {
-            blobClient = getCloudBlockBlob(blobContainer, zipFileName, envelopeId);
-
-            Optional<String> lease = blobManager.acquireLease(blobClient, containerName, envelope.getZipFileName());
+            blob = getCloudBlockBlob(blobContainer, zipFileName, envelopeId);
+            leaseId = blobManager.acquireLease(blob, containerName, envelope.getZipFileName());
 
-            if (lease.isPresent()) {
-                BlobInputStream bis = getBlobInputStream(blobClient, containerName, zipFileName, envelopeId);
+            if (leaseId.isPresent()) {
+                BlobInputStream bis = getBlobInputStream(blob, containerName, zipFileName, envelopeId);
                 ZipFileProcessingResult result = processInputStream(bis, containerName, zipFileName, envelopeId);
 
                 uploadParsedZipFileName(envelope, result.getPdfs());
 
                 envelopeProcessor.handleEvent(envelope, DOC_UPLOADED);
             }
-        } catch (FailedUploadException failure) {
-            log.error(failure.getMessage(), failure.getCause());
```

Comment:
Unnecessary handling with potential maintainability issues - this bit assumes that the exception's message will contain the full description of the situation, not just the local problem. We used to have logging like this in the past and ended up having insufficient information in our logs.